tests/cupy_tests/padding_tests/test_pad.py

```diff
@@ -244,7 +240,7 @@ def test_pad_via_func_returning_vector(self, xp, dtype):
         def _padwithtens(vector, pad_width, iaxis, kwargs):
             vector[:pad_width[0]] = 10
             vector[-pad_width[1]:] = 10
-            return vector  # returning vector required by old NumPy (<=1.12)
+            return vector
```

Comment:
From the comment, I guess the test passes without returning `vector`.

Reply:
Yes it's tested in `test_pad_via_func_modifying_inplace` above.

I thought this test is written specifically for the case `vector` is returned.
But in the NumPy `pad` docs, no return value is mentioned at all.
I think this test can be removed.

Reply:
Removed in b0c7be9be52628d41285c2fb8dbf7b7b125beb6e.
Thanks!